### DISCUSSION TITLE
Ubuntu22.04开启带有访问密码的TCP隧道时报错

### DISCUSSION THREAD

**Discussion by autoreleasepools** [natfrp/wiki#282] [使用问题]
针对枣庄多线2设置了访问密码后，在终端使用frpc开启隧道的时候提示如下错误：
failed to parse proxy ssh, err: local ip or plugin is required
测试了一下没有设置访问密码的枣庄多线4，是可以正常启动的，是我哪里没有配置好吗？

**Comment by autoreleasepools** [comment]
刚才测试了一下枣庄多线4如果有访问密码的话也是可以正常创建隧道的，但是枣庄多线2取消掉访问密码后仍然会提示上述错误，是否时枣庄多线2出现了什么问题？

**Comment by fakeboboliu** [comment]
请确认一下 本地ip 正确设置了
